### DISCUSSION TITLE
None of the security onion commands are found

### DISCUSSION THREAD

**Discussion by AJVenza** [Security-Onion-Solutions/securityonion#12568] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

32gb

### Storage for /

139.4gb

### Storage for /nsm

278.1gb

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

I seem to have an issue with security onion, none of the so-* commands are found at all, for example so-rule is not found, I have re-installed multiple times and it happens every time, soup does work though and says security onion is fully updated.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Are there errors in your `/root/soup.log`?
